### DISCUSSION TITLE
Show the time

### DISCUSSION THREAD

**Discussion by sqq135** [tomasklaen/uosc#863] [Ideas]
Just like the Android player, it will display the time. Can uosc display the current time in the lower part when the timeline appears? Just a idea, I didn't find the right way to show the time.

**Comment by tomasklaen** [comment]
#617

**Comment by sqq135** [comment]
Thanks, I know. It's may not a good idea. I use a very good player on android based on mpv, its ui has the clock. And I hid the status bar of the Windows, so it's not convenient to watch the time when tap the fullscreen shortcut.

And another idea is that show the name of video near the timeline, so I can know all the information when I only need to move the mouse to the lower part. I know it hamper immersion a little. But it is very practical.
